scp/lib/ui/cards.dart

```diff
@@ -23,13 +32,14 @@ Widget appointmentCard(BuildContext context) {
       padding: const EdgeInsets.only(top: 12.0),
       child: InkWell(
         onTap: () async {
-          SharedPreferences prefs = await SharedPreferences.getInstance();
```

Comment:
Don't edit the existing widget, Try creating a fresh widget and replace it in the home page without disturbing the existing card codes.